taworks/taform/views.py

```diff
@@ -500,3 +510,9 @@ def determine_status(df):
         status = df_max.iloc[0]['status']
     app_status = 'Open' if status else 'Closed' 
     return (status_date, status, app_status)
+
+def authenticated(request):
```

Comment:
nice job moving this out and creating a method since it's being used multiple times now.